backend/app/api/endpoints/internal/bots.py

```
@@ -0,0 +1,181 @@
+        )
+
+    except Exception as e:
+        logger.exception(
+            "[internal/bots] Failed to get MCP servers for %s/%s",
+            namespace,
+            bot_name,
+        )
+        raise HTTPException(
+            status_code=500,
+            detail=f"Failed to query MCP servers: {str(e)}",
+        )
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)